js/packages/contexts/styles/app-dimensions.provider.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { createContext, useContext, useState } from 'react'
```

Comment:
please
- rename *.provider.tsx to *.context.tsx
- move app-dimensions.context.tsx to `js/packages/contexts`